force-app/main/default/classes/DonationHistoryController.cls

```diff
@@ -73,11 +77,37 @@ public with sharing class DonationHistoryController {
             throw new AuraHandledException(ex.getMessage());
         }
     }
+    
+    /** 
+    * @author Salesforce.org
+    * @date 2021
+    * @description method to return whether or not Payments are enabled in org
+    * @return boolean isPaymentEnabled
+    */
+    @AuraEnabled(cacheable=true)
+    public static boolean arePaymentsEnabled(){
+        npe01__Contacts_And_Orgs_Settings__c settings = npe01__Contacts_And_Orgs_Settings__c.getOrgDefaults();
+		boolean isPaymentEnabled = settings.npe01__Payments_Enabled__c;
+		return isPaymentEnabled;
+    }
 
-    private static List<DonationHistoryView> buildHistoryFromOCRs(List<Opportunity> contactRoles) {
+    /** 
+    * @author Salesforce.org
+    * @date 2021
+    * @description method creates list of DonationHistoryView to return to LWC
+    * @return List<DonationHistoryView>
+    */
+    private static List<DonationHistoryView> buildHistoryFromOCRs( List<Opportunity> contactRoles) {
+        boolean isPaymentEnabled = arePaymentsEnabled();
         List<DonationHistoryView> donationHistories = new List<DonationHistoryView>();
-        for(Opportunity role : contactRoles) {
-            donationHistories.add(new DonationHistoryView(role));
+        if(isPaymentEnabled){
+            for(Opportunity role : contactRoles) {
+            	donationHistories.add(new DonationHistoryView(role, role.npe01__OppPayment__r.get(0).npe01__Payment_Method__c));
+        	}
+        }else{
+            for(Opportunity role : contactRoles) {
+            	donationHistories.add(new DonationHistoryView(role, EMPTY_PAYMENT_METHOD));
+        	}
```

Comment:
this could be a lot easier!

for (Opportunity opp : opportunities) {
    donationHistories.add(new DonationHistoryView(opp, (isPaymentEnabled) ? role.npe01__OppPayment__r.get(0).npe01__Payment_Method__c) : EMPTY_PAYMENT_METHOD);
}

Reply:
agreed